onnxruntime/core/providers/openvino/openvino_execution_provider.h

```diff
@@ -74,14 +74,15 @@ struct OpenVINOExecutionProviderInfo {
   bool disable_dynamic_shapes_{false};
   bool export_ep_ctx_blob_{false};
   bool enable_qdq_optimizer_{false};
+  bool disable_cpu_fallback_{false};
 
   OpenVINOExecutionProviderInfo() = delete;
 
   explicit OpenVINOExecutionProviderInfo(std::string dev_type, std::string precision, bool enable_npu_fast_compile,
                                          size_t num_of_threads, std::string cache_dir, std::string model_priority,
```

Comment:
@preetha-intel 
I'd recommend to use below, its better to pass a reference or a pointer to avoid any copies -
All the std::string function arguments must be `std::string& <var_name>` if we are modifying the string contents in the code definition or if we are not modifying the string it should be `const std::string& <var_name>`
```suggestion
                                         size_t num_of_threads, std::string& cache_dir, std::string& model_priority, .....
```